src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import React from 'react';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+}
+
+export const Tabs: React.FC<Props> = (props) => {
+  const { tabs, selectedTabId, onTabSelected } = props;
+  const selectedTab = tabs.find(
+    (tab: { id: string; }) => tab.id === selectedTabId,
```

Comment:
you already have a type for that
```suggestion
    (tab: Tab) => tab.id === selectedTabId,
```

Reply:
sry for so long, just found that this task was red :( fixes are done, check pls